### PR TITLE
USWDS - Prefix/Suffix: Error on HTML build

### LINKED ISSUE #5309: USWDS - Bug: Input prefix error on HTML compile
There's a current bug on build with `npm run build:html`. Issue seems to be missing closing tag. Needs to be fixed so as to not break templates on site.

### PR DESCRIPTION
**Pull request by mejiaj**
# Summary

**Fixed input prefix/suffix templates.** This PR does the following:
- Adds a missing closing tag on input prefix. 
- Sets default argument for `disabled_state` in StorybookJS.
- CI tests now check for successful HTML build (via `npm run build:html`).

## Breaking change

This is not a breaking change.

## Related issue

Closes #5309.

## Related pull requests

Unsure if changelog update is required.

## Preview link

Preview link:

## Problem statement

Input prefix was causing a HTML build error because of a missing closing tag.

<details><summary>HTML build error</summary>
<p></p>

```bash
❯ npm run build:html

> @uswds/uswds@3.4.1 build:html
> webpack build --config ./webpack.twig.config.js && npm run prettier:html

ERROR in   Error: Parse Error: <input type="text" id="example-input-prefix-error" class="usa-input" pattern="[0-9]*" inp  utmode="numeric" </div>
```
</details> 

## Solution

1. Added closing tag to input prefix [018df5a].
2. Set default arg for `disabled_state` in story [431805a].
3. Updated CI tests to check for HTML build errors in the future [da51d88].

## Major changes

Only major change is NPM script `test:ci` now checks for HTML build issues.

## Testing and review

1. On `develop` run `build:html` to confirm error.
2. On this branch, run `build:html` to confirm the fix.
3. There should be no regression to input prefix story.

### CODE REVIEW

File: packages/usa-input-prefix-suffix/src/usa-input-prefix.twig [hunk 1]
```diff
@@ -31,6 +31,7 @@
       class="usa-input"
       pattern="[0-9]*"
       inputmode="numeric"
+    >
```
Comment (thisisdano) [inline packages/usa-input-prefix-suffix/src/usa-input-prefix.twig:34]: ✨

**Review by thisisdano — APPROVED** [review]
lgtm

**Review by amyleadem — APPROVED** [review]
Looks good to me. 
- [x] Confirmed `build:html` error on `develop`
- [x] Confirmed no errors during `build:html` on this branch
- [x] Confirmed  that there are no regressions in the input prefix story

**Review by mahoneycm — APPROVED** [review]
Working great! Thanks for fixing those disabled default controls!

**Pull request merged**